vcnc-server/vcnc-sampler/app.js

```diff
@@ -87,31 +84,51 @@ function requestHandler(request, response) {
   response.end();
 }
 
-const server = http.createServer(requestHandler);
-server.setTimeout(5000, () => {
-});
+// Create server
+//
+let server;
```

Comment:
Sure.  And when you do, you can move it outside... ;-)

But seriously:  When something is outside the function, the reader wonders why it's outside, and then starts looking for why it's outside and can't find it.  It's a distraction.   Worse, since it is "let" and not "const", you're saying that not only is this variable visible outside (and now I have to wonder why) you're saying that something _outside_ can _change it_!!!  All kinds of alarms go off.

But if it's _inside_ and _const_, then it's boring... there's nothing that can go wrong, nothing to investigate further.  Easy.  Saves everybody time.  Prevents bugs.

That's why I'm gently insisting that you move it inside and make it const.  Please see the two principles above.  Thanks!